### DISCUSSION TITLE
Perfis e listagem de perfis

### DISCUSSION THREAD

**Discussion by highlabs** [retromeet/core#26] [Ideas]
Olar pessoas.

Você tem alguma ideia de como vai trabalhar os Perfis e listagem de perfis? Pergunto por motivos de: Eu odeio a ideia de deslizar para "like" e "dislike", como foi amplamente difundido pelo Tinder.

#### Listagem de perfis

Antigamente, antes do OkCupid ser comprado pele Tinder, ele havia um "catalogo de perfis", onde você aplicava os filtros (localidade, gênero, aparência, etc) e ele listava as pessoas disponíveis dentro daquele filtro. Isso diminuia a ideia das pessoas serem tratadas como "pedaços de carne". Além de evitar aquela "urgência" de ver outros perfis e permite as pessoas se dedicarem melhor ao perfis uma das outras.

#### Perfis

Já faz algum tempo que as pessoas estão saindo dos apps de encontro e criando os "date me docs":

- https://www.wired.com/story/date-me-google-docs-and-the-hyper-optimized-quest-for-love/
- https://parade.com/living/date-me-docs

Se assemelhando bastante com os antigos "classificados". 🧓🏾 Seria interessante dar a oportunidade das pessoas deixarem seus perfis públicos ou não.

**Comment by renatolond** [comment]
É, a minha ideia é ir pra uma ideia nessa de listagem de perfis baseado em filtros mesmo. Minha ideia é que o usuário possa marcar todas as partes do seu perfil com diferentes níveis de privacidade, tipo como é o caso com outros cantos do fediverso, mas no caso do retromeet as privacidades seriam um pouco diferentes: "visível pra todos", "visível pra quem eu marquei como interessante/favoritei" e "visível só pra quem deu match" e no caso do match seria só se marcar como favorito/interessante mutuamente.

Além disso, numa outra linha, a ideia seria também poder marcar partes do perfis como "NSFW" (embora a nomenclatura não seja ideal), de novo, permitindo conteúdo mais sexual mas que passaria como um certo aviso de conteúdo.

A ideia de deslizar é exatamente o design que eu quero evitar :)
